### PR TITLE
Added r-ISLR2 as a pacakge

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/r @conda-forge/staged-recipes ready for review!

File: recipes/r-islr2/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+mv DESCRIPTION DESCRIPTION.old
+grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (mfansler) [inline recipes/r-islr2/build.sh:6]: I think the `Priority` is not relevant anymore.

File: recipes/r-islr2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ISLR2_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ISLR2/ISLR2_{{ version }}.tar.gz
+  sha256: e64d6d20d6745c58621842bc22d1553fc45edaa390834ba757bdd16c07bfacc2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
```
Comment (mfansler) [inline recipes/r-islr2/meta.yaml:19]: This can be `noarch`.

```yaml
  noarch: generic
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for the submission. All suggested changes would have already been implemented if the latest [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) were used to generate the recipe. Consider using that for all future CRAN package submissions.

File: recipes/r-islr2/build.sh (hunk 1 above)
Comment (thewchan) [reply recipes/r-islr2/build.sh:6]: Do you mean I should remove line 6, or?

File: recipes/r-islr2/meta.yaml (hunk 2 above)
Comment (thewchan) [reply recipes/r-islr2/meta.yaml:19]: ✅

File: recipes/r-islr2/build.sh (hunk 1 above)
Comment (mfansler) [reply recipes/r-islr2/build.sh:6]: Both 5 and 6 can be dropped.

Comment (thewchan) [reply recipes/r-islr2/build.sh]: ✅

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
@conda-forge/staged-recipes looks ready to merge

**Pull request merged**